packages/rancher-monitoring/rancher-monitoring/package.yaml

```diff
@@ -1,7 +1,7 @@
 url: https://github.com/prometheus-community/helm-charts.git
 subdirectory: charts/kube-prometheus-stack
 commit: 7c6906ca223344c06952007fda670c6c81e6d1da
-version: 103.2.1
+version: 103.2.2
```

Comment:
In case there are issues during QA testing, should mark this as an rc, so it doesn't accidentally get shipped with bugs/regressions in it.

I'll leave it up to you if you think it's worth the additional overhead of un-rcing for the release